clusterloader2/pkg/prometheus/experimental.go

```diff
@@ -128,6 +128,7 @@ func (pc *Controller) snapshotPrometheusDiskIfEnabled() error {
 		snapshotRetryTimeout,
 		func() (bool, error) {
 			err := pc.trySnapshotPrometheusDisk(pc.diskMetadata.name, snapshotName, pc.diskMetadata.zone)
+			klog.Error(err)
```

Comment:
I would prefer sth like:
```
if err != nil {
  klog.Errorf("Trying to snapshot prometheus disk failed: %v", err)
}
```

it's more informative

Reply:
Done